lib/recomb_map.c

```diff
@@ -228,6 +241,16 @@ recomb_map_sample_poisson(recomb_map_t *self, gsl_rng *rng, double start)
     return recomb_map_shift_by_mass(self, left_bound, mass_to_next_recomb);
 }
 
+void
+recomb_map_change_rates(recomb_map_t *self,
```

Comment:
I guess to make this as readable as we can, we should change this to ``recomb_map_convert_rates``. Then, in msprime.h we do 
```
typedef msp_convert_func (double convert (void *obj, double rate))
```
(Or whatever it is; I can never remember the syntax). Then we have 
```
void
￼recomb_map_change_rates(recomb_map_t *self, msp_convert_func convert, void *obj)
```